exporter/prometheusremotewriteexporter/wal.go

```diff
@@ -324,113 +332,81 @@ func (prwe *prweWAL) persistToWAL(requests []*prompb.WriteRequest) error {
 			return err
 		}
 		wIndex := prwe.wWALIndex.Add(1)
+		prwe.log.Debug("write", zap.Uint64("index", wIndex))
```

Comment:
Do we still need it or can we set it to trace level? (in that case, it should be a bit more explicit)

Reply:
turns out – there is no `trace` level in zap ... so we need to either leave it as debug or remove entirely

Reply:
how is the full log message displayed?

Reply:
```
=== RUN   TestWAL_E2E
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:364	read	{"index": 1}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:390	wal empty - waiting for write
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:335	write	{"index": 1}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:335	write	{"index": 2}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:335	write	{"index": 3}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:364	read	{"index": 1}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:364	read	{"index": 2}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:364	read	{"index": 3}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:364	read	{"index": 4}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:390	wal empty - waiting for write
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:364	read	{"index": 4}
2023-05-25T16:28:08.513+0200	DEBUG	prometheusremotewriteexporter/wal.go:390	wal empty - waiting for write
```